app/components/dex-indicator.jsx

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux';
+
+export function DexIndicator ({ dex, profile }) {
+
+  if ((dex && !dex.shiny) || profile) {
```

Comment:
right, that's why `|| profile` is necessary - i want this expression to evaluate to true if we're on the `profile`. the logic is: render nothing if i'm a dex and the dex is not shiny, or if i'm on the profile page (because this component is used in the `HeaderComponent` which is used on both the profile page and the dex page, but in that case i only want it to show on the dex page). is there a better way to write that out?